cmd/podman/network_create.go

```diff
@@ -0,0 +1,88 @@
+// +build !remoteclient
+
+package main
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/adapter"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+/*
+       --attachable           Enable manual container attachment
+       --aux-address map      Auxiliary IPv4 or IPv6 addresses used by Network driver (default map[])
+       --config-from string   The network from which copying the configuration
+       --config-only          Create a configuration only network
+   -d, --driver string        Driver to manage the Network (default "bridge")
+       --gateway strings      IPv4 or IPv6 Gateway for the master subnet
+       --ingress              Create swarm routing-mesh network
+       --internal             Restrict external access to the network
+       --ip-range strings     Allocate container ip from a sub-range
+       --ipam-driver string   IP Address Management Driver (default "default")
+       --ipam-opt map         Set IPAM driver specific options (default map[])
+       --ipv6                 Enable IPv6 networking
+       --label list           Set metadata on a network
+   -o, --opt map              Set driver specific options (default map[])
+       --scope string         Control the network's scope
+       --subnet strings       Subnet in CIDR format that represents a network segment
+*/
+
+var (
+	networkcreateCommand     cliconfig.NetworkCreateValues
+	networkcreateDescription = `create network`
+	_networkcreateCommand    = &cobra.Command{
+		Use:   "create [flags] [NETWORK]",
+		Short: "network create",
+		Long:  networkcreateDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			networkcreateCommand.InputArgs = args
+			networkcreateCommand.GlobalFlags = MainGlobalOpts
+			networkcreateCommand.Remote = remoteclient
+			return networkcreateCmd(&networkcreateCommand)
+		},
+		Example: `podman create podman1`,
```

Comment:
would be nice to have two more examples with the most common options or the less intuitive ones thrown in.  Perhaps --label, --config-from-string and a few others?

Reply:
Oh, I see, not yet supported for many.